azure-pipelines.yml

```diff
@@ -1,19 +1,20 @@
 jobs:
-- job: BuildLinuxWithSonarCloud
-  displayName: Build for Linux\CMake With SonarCloud
+- job: BuildLinuxWithSonarQube
+  displayName: Build for Linux\Autotools With SonarQube
   pool:
     vmImage: 'ubuntu-latest'
   variables:
-    BUILD_WRAPPER_DOWNLOAD_URL: https://sonarcloud.io/static/cpp/build-wrapper-linux-x86.zip
+    BUILD_WRAPPER_DOWNLOAD_URL: <SONARQUBE_HOST_URL>/static/cpp/build-wrapper-linux-x86.zip # Replace <SONARQUBE_HOST_URL> with your SonarQube server URL
```

Comment:
I mean to have SONARQUBE_HOST_URL as the variable. 
And then in download use ${{variables.SONARQUBE_HOST_URL}}/static/cpp/build-wrapper-linux-x86.zip

Reply:
I made the change, thanks!